### PR TITLE
clk: fixes in header files inline description comments 

### PR DESCRIPTION
**Pull request by etienne-lms**

### CODE REVIEW

File: core/include/drivers/clk_dt.h [hunk 1]
```diff
@@ -40,6 +40,7 @@
  *
  * Return TEE_SUCCESS in case of success
  * Return TEE_ERROR_DEFER_DRIVER_INIT if clock is not initialized
+ * Return TEE_ERROR_ITEM_NOT_FOUND if the DT does provide a clock reference
```
Comment (jforissier) [inline core/include/drivers/clk_dt.h:43]: "does" or "does not"?

Comment (etienne-lms) [reply core/include/drivers/clk_dt.h:43]: indeed, thanks.

**Comment by etienne-lms** [comment]
comment addressed

**Review by jforissier — COMMENTED** [review]
`Acked-by: Jerome Forissier <jerome.forissier@linaro.org>`

**Comment by etienne-lms** [comment]
tag applied

**Comment by etienne-lms** [comment]
merge in 3.20.0? Not mandatory, just typos fixing.

**Pull request merged**